systers_portal/meetup/forms.py

```diff
@@ -63,12 +66,14 @@ class AddMeetupForm(ModelFormWithHelper):
     class Meta:
         model = Meetup
         fields = ('title', 'slug', 'date', 'time', 'meetup_location', 'venue', 'description',
-                  'meetup_picture')
+                  'resources')
         widgets = {'date': forms.DateInput(attrs={'type': 'text', 'class': 'datepicker'}),
                    'time': forms.TimeInput(attrs={'type': 'text', 'class': 'timepicker'})}
         helper_class = SubmitCancelFormHelper
         helper_cancel_href = "{% url 'index' %}"
 
+    images = MultiFileField(required=False)
```

Comment:
Apologies if you have already gone through this

But why can't you just use https://docs.djangoproject.com/en/3.0/topics/http/file-uploads/#uploading-multiple-files

Reply:
Using the above actuallly requires changing the views.py which was too much work, using the way the library works, it was much easier to do it without changing the views or the forms much. Because the way the views are written rn, it becomes really very difficult to make even a minor change to them

Reply:
Got you!